modules/lang-painless/src/main/java/org/elasticsearch/painless/antlr/EnhancedPainlessLexer.java

```diff
@@ -67,19 +67,12 @@ protected boolean isSlashRegex() {
         if (lastToken == null) {
             return true;
         }
-        switch (lastToken.getType()) {
-            case PainlessLexer.RBRACE:
-            case PainlessLexer.RP:
-            case PainlessLexer.OCTAL:
-            case PainlessLexer.HEX:
-            case PainlessLexer.INTEGER:
-            case PainlessLexer.DECIMAL:
-            case PainlessLexer.ID:
-            case PainlessLexer.DOTINTEGER:
-            case PainlessLexer.DOTID:
-                return false;
-            default:
-                return true;
-        }
+        return switch (lastToken.getType()) {
+            // tag::noformat
+            case PainlessLexer.RBRACE, PainlessLexer.RP, PainlessLexer.OCTAL, PainlessLexer.HEX, PainlessLexer.INTEGER,
+                 PainlessLexer.DECIMAL, PainlessLexer.ID, PainlessLexer.DOTINTEGER, PainlessLexer.DOTID -> false;
```

Comment:
I'm not convinced this is more legible, actually. Before, with ease case on a line, you would see any changes in a diff very easily. We could (and should) have sorted the lines alphabetically too. What do you think about reverting this change?

Reply:
Agreed! I've reverted this change.